backend/app/requirements/main.txt

```diff
@@ -3,6 +3,7 @@ Click==7.0
 fastapi==0.45.0
 gunicorn==20.0.4
 h11==0.8.1
+httptools==0.1.1
```

Comment:
Some explanation: yes, it's intentional because it was also before installed in a broken conflicting way 🤯  with old pip. If we look at old build pipelines it's visible as a non breaking warning.
New released pip 20.3 revealed this issue and breaks during installing.

Reply:
Details here from the build of second commit: https://app.circleci.com/pipelines/github/openstax/output-producer-service/920/workflows/68ed078e-e6c5-4695-af77-bf02b3e42eb6/jobs/3887